### DISCUSSION TITLE
posts/game/2024/11/16/01/%EC%A4%91%EC%B9%B4-%EB%A7%88%EC%97%A0%EB%B8%94/

### DISCUSSION THREAD

**Discussion by giscus** [melted-mint/fuwari-edited#4] [Comments]
# posts/game/2024/11/16/01/%EC%A4%91%EC%B9%B4-%EB%A7%88%EC%97%A0%EB%B8%94/

히히

https://meltedmint.com/posts/game/2024/11/16/01/%EC%A4%91%EC%B9%B4-%EB%A7%88%EC%97%A0%EB%B8%94/

**Comment by applemint35** [comment]
응애 뉴비에오
